aws-predictions/src/main/java/com/amplifyframework/predictions/aws/http/LivenessWebSocket.kt

```diff
@@ -493,7 +493,7 @@ internal class LivenessWebSocket(
     private fun isTimeDiffSafe(diffInMillis: Long) = kotlin.math.abs(diffInMillis) < FOUR_MINUTES
 
     companion object {
-        private const val NORMAL_SOCKET_CLOSURE_STATUS_CODE = 1000
+        internal const val NORMAL_SOCKET_CLOSURE_STATUS_CODE = 1000
```

Comment:
Suggestion: Should we keep access modifier unchanged? destroy() already has default value for parameter reasonCode= NORMAL_SOCKET_CLOSURE_STATUS_CODE, so we might not need to pass this value from caller site when reason code is null.